src/main/java/org/jenkinsci/test/acceptance/po/FreeStyleMultiBranchJob.java

```diff
@@ -14,4 +15,28 @@
     public FreeStyleMultiBranchJob(Injector injector, URL url, String name) {
         super(injector, url, name);
     }
+
+    @Override
+    public <T extends BuildStep> T addBuildStep(Class<T> type) {
+        return addStep(type, "builder");
+    }
+
+    @Override
+    public <T extends PostBuildStep> T addPublisher(Class<T> type) {
+        T p = addStep(type, "publisher");
+
+        publishers.add(p);
+        return p;
+    }
+
+    private <T extends Step> T addStep(Class<T> type, String section) {
```

Comment:
@olivergondza Unfortunately the FreeStyleMultiBranchJob' s UI mixes the standard hetero-list with another one with other path only for publishers and builder, so overriding the `Job#addStep` method results in other parts of the test failing, I need to separate methods